tools/clangrttool/clangrttool_unittest.py

```diff
@@ -0,0 +1,70 @@
+import os
+import unittest
+
+from tools.clangrttool.clangrttool import normalize_clang_lip_path, ClangRuntimeToolError
+
+clang_lib_path_common = "Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/14.0.0/lib/darwin"
+xcode_developer_dir_default = "/Applications/Xcode.app/Contents/Developer"
```

Comment:
Can we add one that is named something different, like `ProjectBuilder.app`?

Reply:
See line 12 below.